gnmvidispine/vs_item.py

```diff
@@ -420,7 +421,7 @@ def copyToPlaceholder(self,host='localhost',port=8080,user='admin',passwd=None):
         :param passwd: password for the VS server to copy to (default: None)
         :return: a new VSItem representing the duplicated item
         """
-        md = self.metadata_document()
+        md = str(self.metadata_document())
```

Comment:
why do we need `str()` here? is `self.metadata_document()` returning a `bytestring` maybe? pls check and encode/decode appropriately

Reply:
I expect it is written like that because it works under Python 2 and 3.

Python 2 and 3 handle encode and decode differently. Best to leave this working.

Reply:
:+1: